app/views/devices_guidance/preparing_chromebooks.md

```diff
@@ -1,47 +1,80 @@
+is available from the National Cyber Security Centre.
 
-## How to enrol Chromebooks
+## Getting a Google domain
 
-You need to enrol your Chromebooks so you can manage them remotely. Follow the appropriate guidance below for your devices:
+When you ordered your Google Chromebooks you provided your Google domain, which 
+we used to link to your licenses for the Google Admin console.
 
-* [User login required](/devices/enrol-chromebooks-with-user-logins) – suitable for users linked to schools
-* [No user login required](/devices/enrol-chromebooks-without-user-logins) – suitable for users linked to local authorities (for example, those in social care)
+If you didn’t enter a Google domain and don’t have one, you can set one up. Schools can
+set up a Google Domain and get access to Google for Education by [applying for government-funded support](/digital-platforms).
 
-## Getting a Google domain
+For Chromebooks for children linked to schools, local authorities should use the school’s 
+domain so that children can access their remote education as well as social care. If the 
+Chromebook is for a young person who is not linked to a school, local authorities can 
+[set up a Google domain for care leavers and families with children that have a social worker](/devices/google-domain-for-care-leavers-and-children-with-social-worker).
+
+## How to enrol Chromebooks
+
+You need to enrol your Chromebooks so you can manage them remotely.
 
-When you ordered your Google Chromebooks you provided your Google domain, which we used to link to your licenses for the Google Admin console.
+Before you enrol your devices, please ensure that you have the correct number of Chrome 
+licenses available in your domain by going to <span class="app-no-wrap">‘Devices’ > ‘Chrome devices’</span> and clicking the 
+‘Upgrades’ button in the top right. If you do not have enough licenses for the number of devices
+you need to enrol and manage, please contact [COVID.TECHNOLOGY@education.gov.uk](mailto:COVID.TECHNOLOGY@education.gov.uk).
 
-If you didn’t enter a Google domain and don’t have one, you can set one up. Schools can set up a Google Domain and get access to Google for Education by [applying for government-funded support](/digital-platforms).
+First, create an organisational unit called ‘DfE Devices’ in your Google Admin Console, then 
+follow these instructions to enrol your devices:
 
-For Chromebooks for children linked to schools, local authorities should use the school’s domain so that children can access their remote education as well as social care. If the Chromebook is for a young person who is not linked to a school, local authorities can [set up a Google domain for care leavers and families with children that have a social worker](/devices/google-domain-for-care-leavers-and-children-with-social-worker).
+1. Unbox multiple devices and turn on
```

Comment:
It's markdown to you can just use `1.` for each of the bullets and it will auto-number.

Reply:
Annoyingly, RubyMine changes them to the actual number which I don't like either.